### PR TITLE
Add Illumina sample sequencing metrics admin view table

### PR DESCRIPTION
**Pull request by diitaz93**
## Description

Add table **Illumina Sample Sequencing Metrics** to StatusDB with the same columns as the **Sample Lane Sequencing Metrics** table and the same operations (search and sort).

### Added

- `IlluminaSampleSequencingMetricsView` model

### How to prepare for test

- [x] Ssh to relevant server (depending on type of change) or deploy locally with script

### How to test

- [x] Do deploy in the VM or locally

### Expected test outcome

- [x] Check that the table is rendered correctly, the search and sort operations work
## Review

- [x] Tests executed by SD
- [x] "Merge and deploy" approved by CO
Thanks for filling in who performed the code review and the test!

### This [version](https://semver.org/) is a

- [x] **PATCH** - when you make backwards compatible bug fixes or documentation/instructions

## Implementation Plan

To be merged into a dev branch, will be deployed later

### CODE REVIEW

**Comment by diitaz93** [comment]
## Tests on local web API
Searchable and sortable works, links for flow cells and sample work
Screenshot 2024-05-24 at 12 19 46

File: cg/server/admin.py [hunk 1]
```diff
@@ -640,6 +708,25 @@ class UserView(BaseView):
...
 
+class IlluminaSampleSequencingMetricsView(BaseView):
+    column_list = [
+        "flow_cell",
+        "sample",
+        "flow_cell_lane",
+        "total_reads_in_lane",
+        "base_passing_q30_percent",
+        "base_mean_quality_score",
+        "_yield",
```
Comment (ChrOertlin) [inline cg/server/admin.py:719]:
```suggestion
        "yield",
```
Does this pick names from the DB table directly? In that case the name is just `yield`

File: cg/server/admin.py [hunk 2]
```diff
@@ -441,6 +447,68 @@ class AnalysisView(BaseView):
...
+    column_sortable_list = [
+        "sequenced_at",
+        "sequencer_type",
+        "sequencer_name",
+        "data_availability",
+        "has_backup",
+        "sequencing_started_at",
+        "sequencing_completed_at",
+        "demultiplexing_started_at",
+        "demultiplexing_completed_at",
```
Comment (ChrOertlin) [inline cg/server/admin.py:492]: Once I know if we can get accurate dates here we can keep this..

Also, if we have these 4 time stamps. What purpose does the `sequenced_at` serve?

Comment (diitaz93) [reply cg/server/admin.py:492]: fixed in source branch

File: cg/server/admin.py (hunk 1 above)
Comment (diitaz93) [reply cg/server/admin.py:719]: I am not sure, but if it doesn't I have to change this to `yield_` anyway. I will test this when the source branch is merged

**Review by ChrOertlin — APPROVED** [review]
With the new changes, looks good!

File: cg/server/admin.py (hunk 1 above)
Comment (diitaz93) [reply cg/server/admin.py:719]: fixed to `yield_`

**Pull request merged**